Source/JavaScriptCore/runtime/TemporalPlainYearMonth.h

```diff
@@ -57,6 +57,12 @@ class TemporalPlainYearMonth final : public JSNonFinalObject {
 
     String monthCode() const;
 
+    String toString(JSGlobalObject*, JSValue options) const;
+    String toString() const
+    {
+        return ISO8601::temporalYearMonthToString(m_plainYearMonth, ""_s);
+    }
```

Comment:
Does this function body need to be in the header? Why not put it in the `.cpp` file instead?